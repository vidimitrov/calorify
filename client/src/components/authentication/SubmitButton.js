import styled from 'styled-components';

const SubmitButton = styled.input`
  background-color: #e5c769;
  border: none;
  color: white;
  padding: 15px 80px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  text-transform: uppercase;
  font-size: 13px;
  -webkit-box-shadow: 0 10px 30px 0 rgba(95,186,233,0.4);
  box-shadow: 0 10px 30px 0 rgba(95,186,233,0.4);
  -webkit-border-radius: 5px 5px 5px 5px;
  border-radius: 5px 5px 5px 5px;
  margin: 5px 20px 20px 20px;
  -webkit-transition: all 0.3s ease-in-out;
  -moz-transition: all 0.3s ease-in-out;
  -ms-transition: all 0.3s ease-in-out;
  -o-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
`;

export default SubmitButton;
