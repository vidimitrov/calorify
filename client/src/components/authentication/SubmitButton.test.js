import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import SubmitButton from './SubmitButton';

it('Authentication/SubmitButton component renders correctly', () => {
  const tree = renderer.create(
    <SubmitButton
      disabled={false}
      onClick={() => { }}
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
