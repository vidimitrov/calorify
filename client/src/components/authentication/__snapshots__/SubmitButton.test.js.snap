// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication/SubmitButton component renders correctly 1`] = `
.c0 {
  background-color: #e5c769;
  border: none;
  color: white;
  padding: 15px 80px;
  text-align: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
  text-transform: uppercase;
  font-size: 13px;
  -webkit-box-shadow: 0 10px 30px 0 rgba(233,193,95,0.4);
  box-shadow: 0 10px 30px 0 rgba(233,193,95,0.4);
  -webkit-border-radius: 5px 5px 5px 5px;
  border-radius: 5px 5px 5px 5px;
  margin: 5px 20px 20px 20px;
  -webkit-transition: all 0.3s ease-in-out;
  -moz-transition: all 0.3s ease-in-out;
  -ms-transition: all 0.3s ease-in-out;
  -o-transition: all 0.3s ease-in-out;
  -webkit-transition: all 0.3s ease-in-out;
  transition: all 0.3s ease-in-out;
}

.c0:disabled {
  background-color: #f6f6f6;
  color: #c2bcbc;
  box-shadow: none;
}

<button
  className="c0"
  disabled={false}
  onClick={[Function]}
/>
`;
